Docs/update user docs (#264)
secvisogram/secvisogram: docs/user/document/distribution/tlp/url-usage.en.md

# Url - Usage

Reference to the meaning of the used TLP labels.

> The default value should be a link to the schema at first.org.

Reviewer: ```suggestion
The default value is a link to the definitions from FIRST.
```
Add link here.

Author: Added a link to https://www.first.org/tlp/ - is that what you meant?
